_docs/objects/pickable.md

```diff
@@ -53,13 +51,23 @@ how to configure physics layers for Godot XR Tools.
 
 ### XRToolsPickable
 
-| Property           | Description                                                     |
-| ------------------ | --------------------------------------------------------------- |
-| Press To Hold      | If true, the user must hold down the grab button specified in the pickup. |
-| Picked Up Layer    | Physics layer of the object while being held. |
-| Hold Method        | Method used to hold the object in the pickup. |
-| Release Mode       | RigidBody mode to set when the object is released. |
-| Ranged Grab Method | Method used for ranged-grabbing of the object. |
-| Ranged Grab Speed  | Speed to perform ranged-grabbing. |
-| Picked By Exclude  | Optional name of a group this object refuses to be picked up by. |
-| Picked By Require  | Optional name of a group this object requires being picked up by. |
+| Property | Description |
+| ---- | ------------ |
+| Enabled            | If true, the object can be picked up |
```

Comment:
I think its important to clarify here that disabling the object means it can't be picked up, but it still interacts with the world. e.g. it will still fall and hit the ground.

Reply:
Added an extended note to the end of the properties discussing this.